examples/cpp/xtypes/README.md

```diff
@@ -21,7 +21,8 @@ For further information regarding the configuration environment, please refer to
 The particularity of this example resides in the use of X-Types, which allows the definition of types at runtime.
 In this case:
 
-1. The publisher application defines a type at runtime using the Fast DDS Dynamic Types API.
+1. The publisher application creates a type at runtime using the Fast DDS Dynamic Types API.
+The types can be defined through C++ API or XML file.
```

Comment:
It would be helpful to add a link to the XML configuration here
```suggestion
The types can be defined through C++ API or [XML file](https://fast-dds.docs.eprosima.com/en/latest/fastdds/xml_configuration/dynamic_types.html).
```